src/adapters/scene.js

```diff
@@ -190,6 +190,22 @@ Scene.prototype.initMapBox = function() {
   listen('move_mobile_geolocation_button', bottom => {
     this.moveMobileGeolocationButton(bottom);
   });
+
+  listen('show_mobile_geolocation_button', () => {
```

Comment:
good idea,
I'm not going to do the "body classes" approach now, because the future animations enhancements (QMAPS-1452) will surely require events to make the transition depend on the exact height where the panel is being dragged with the finger, and this height will most probably be passed by an event